assets/messages.yml

```diff
@@ -1470,6 +1470,7 @@ messages:
         -- [Minecraft Launcher|https://bugs.mojang.com/browse/MCL] --- Bugs about the Minecraft Launcher specifically
         -- [Minecraft (Bedrock codebase)|https://bugs.mojang.com/browse/MCPE] --- Android, iOS, Windows 10 (from the Microsoft Store), Xbox, Nintendo Switch, PlayStation, Fire OS, and Gear VR
         -- [Minecraft Dungeons|https://bugs.mojang.com/browse/MCD] --- Action-adventure title set in the Minecraft universe
+        -- [Minecraft Realms|https://bugs.mojang.com/browse/REALMS] --- Bugs related to or caused by Realms and not an issue with Minecraft itself
```

Comment:
What do you think about something like this?
> Bugs which only occur on Realms but not in singleplayer or on third-party servers

Otherwise it might be a bit difficult for the user to tell what an issue with Minecraft itself is.

Reply:
Sounds good. If you want to go ahead and commit the change feel free, I won't be able to get to it until tomorrow since I'm on mobile rn and not in a position where I can do it - but anyway, seems like better wording to me for sure

Reply:
Thanks, have adjusted it and added it to the `wrong-project` message as well. Might be good if someone else working actively on the Realms project would have a look as well.